### PR TITLE
Added order page and test

### PR DESCRIPTION
**Pull request by chenasrika**
Hi Pushpesh
Can you approve this PR
Thanks
Chena

### CODE REVIEW

File: src/Pages/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package Pages;
+
+public class OrderPage {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		System.out.println("Hello from OrderPage");
+
```
Comment (PushpeshKumar26) [inline src/Pages/OrderPage.java:10]: Hi, where is the logic

File: src/tests/OrderPageTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package tests;
+
+public class OrderPageTest {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		System.out.println("Hello from OrderPageTest");
+
```
Comment (PushpeshKumar26) [inline src/tests/OrderPageTest.java:12]: Hi where is the assertion

**Review by PushpeshKumar26 — CHANGES_REQUESTED** [review]
Hi, Can you check the review comments
Thanks
Pushpesh

File: src/tests/OrderPageTest.java (hunk 2 above)
Comment (chenasrika) [reply src/tests/OrderPageTest.java:12]: Added

File: src/Pages/OrderPage.java (hunk 1 above)
Comment (chenasrika) [reply src/Pages/OrderPage.java:10]: Added

**Review by PushpeshKumar26 — APPROVED** [review]

**Pull request merged**